src/tests/parser_test.py

```diff
@@ -416,6 +417,46 @@ def test_raise_from():
     parser.parse(code, 'file_path')
 
 
+@pytest.mark.skipif(six.PY2, reason='Matrix multiplication operator is '
+                                    'invalid in Python 2.x')
+def test_simple_matrix_multiplication():
+    """Make sure 'a @ b' doesn't trip the parser."""
+    if sys.version_info.minor < 5:
+        return
+    parser = Parser()
+    code = CodeSnippet("""
+        def foo():
+            a @ b
+    """)
+    parser.parse(code, 'file_path')
```

Comment:
Allow me [to quote myself](http://amir.rachum.com/blog/2017/01/14/explicit-tests/):

> The thing is, while this string appears in many tests, it is an arbitrary string, which is coincidentally the same. We gain nothing from refactoring it out into a constant, while losing readability.